.github/workflows/build.yml

```
@@ -73,6 +77,10 @@ on:
         required: false
         default: true
         type: boolean
+      nightly_release:
+        type: boolean
+        default: false
+        required: false
```

For consistency with other inputs

```suggestion
      nightly_release:
        required: false
        default: false
        type: boolean
```